### PR TITLE
added the angular-africa logo

### PR DESCRIPTION
**Pull request by valoni01**
## Checklist for logo submission

- [x] Logo file added to the `logos` folder
- [x] Created a new entry in `metadata.json` (only `name` is mandatory)
- [x] Logo is an SVG with vector paths (no embedded pixel image)
- [x] The logo fills the image and there is no white margin (please remove the white margin if there is any 🙂 )

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -298,6 +298,12 @@
       "description": "Angular Security Lib for OpenID Connect & OAuth2",
       "license": "MIT",
       "website": "https://github.com/damienbod/angular-auth-oidc-client"
+    },
+    "angular-africa.svg": {
+      "creator": "Valentine Awe",
+      "name": "angular-africa",
```
Comment (maartentibau) [inline metadata.json]: Would put `Angular Africa` here

File: metadata.json [hunk 2]
```diff
@@ -298,6 +298,12 @@
...
+      "description": "This is the official logo of the Angular Africa community",
```
Comment (maartentibau) [inline metadata.json]: Maybe change to: `Official logo of the Angular Africa community`

**Comment by fmalcher** [comment]
Thanks for the submission! Can you please make the background transparent and remove the white margin around the logo? Thanks! 🙂 @valoni01

**Review by fmalcher — APPROVED** [review]

**Comment by fmalcher** [comment]
Thank you! @valoni01

**Comment by maartentibau** [comment]
@valoni01 can you rebase your branch with `master` ?

**Comment by maartentibau** [comment]
Thanks @valoni01 🙏

**Pull request merged**